src/main/java/offtop/UserService/listeners/UserDataConsumer.java

```diff
@@ -0,0 +1,31 @@
+package offtop.UserService.Services;
+
+import java.util.Map;
+
+import java.util.*;
+// import org.json.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+import offtop.UserService.Models.User;
+
+@Service
+public class UserDataConsumer {
+    private final Logger logger = LoggerFactory.getLogger(ConsumerService.class);
```

Comment:
This needs to be changed to `    private final Logger logger = LoggerFactory.getLogger(UserDataConsumer.class);
`